specification/metrics/amortizedcost.md

```diff
@@ -0,0 +1,44 @@
+# Amortized Cost
+
+Amortized cost represents the sum of the amortization of upfront fees, the amortization of recurring fees, and the discounted variable rate of the line item. This column is often used to show spending trends that include on demand rates, reservation rates, recurring commitments, as well as amortization of upfront fees. The Fee and Upfront amortization portion of this column’s value should be proportional to the number of units for the line item and the time granularity of the data. The currency for the value specified in Billed Cost can be found in the [Billing Currency](#billingcurrency) column.
+
+The AmortizedCost column MUST be present in the billing data. This column MUST be a numeric value of type Decimal and MUST NOT contain null values. The aggregated Amortized Cost for a billing period WILL NOT match the charge received on the invoice for the same billing period.
+
+
+Practitioners are faced with two main challenges in respect to this metric:
+1. Practitioners need to amortized upfront fees over the duration of the commitment and distribute those funds to the appropriate reporting groups (e.g. tags, resources)
```

Comment:
Changed tense of verb to match sentence
Changed funds to fees to match earlier term
```suggestion
1. Practitioners need to amortize upfront fees over the duration of the commitment and distribute those fees to the appropriate reporting groups (e.g. tags, resources)
```